api/pkg/transformer/pipeline/upi_autoloading_op_test.go

```diff
@@ -0,0 +1,842 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/antonmedv/expr/vm"
+	upiv1 "github.com/caraml-dev/universal-prediction-interface/gen/go/grpc/caraml/upi/v1"
+	"github.com/gojek/merlin/pkg/transformer/jsonpath"
+	"github.com/gojek/merlin/pkg/transformer/symbol"
+	"github.com/gojek/merlin/pkg/transformer/types"
+	"github.com/gojek/merlin/pkg/transformer/types/expression"
+	"github.com/gojek/merlin/pkg/transformer/types/series"
+	"github.com/gojek/merlin/pkg/transformer/types/table"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+)
+
+func TestUPIAutoloadingOp_Execute(t *testing.T) {
```

Comment:
Can we add benchmark to measure how fast is autoloading process?

Reply:
sure I'll add one